src/nomad_material_processing/vapor_deposition/general.py

```diff
@@ -642,6 +642,13 @@ class VaporDepositionStep(ActivityStep):
             component='BoolEditQuantity',
         ),
     )
+    step_index = Quantity(
+        type=str,
+        description='The sequential index of the step in the growth process',
+        a_eln={
+            'component': 'StringEditQuantity',
+        },
+    )
```

Comment:
Lists are ordered in python so they already have an index, or is there another reason you need this? If we add this we would need to modify the normalize method in one of the parents which calculates the start and end times.